sympy/physics/mechanics/system.py

```diff
@@ -758,7 +766,7 @@ def add_joints(self, *joints):
             bodies.update((joint.parent, joint.child))
         coordinates = coordinates.difference(self.q)
         speeds = speeds.difference(self.u)
-        kdes = kdes.difference(self.kdes)
+        kdes = kdes.difference(self.kdes[:] + (-self.kdes)[:])
```

Comment:
Is this the best way to create copies of `self.kdes`?

Reply:
Not sure, this is one of the shortest methods in terms of number of characters.

Reply:
I think using `.copy()` would signify intent better.

Reply:
This is not to create a copy here, but to convert it to a list.